Robust.UnitTesting/Shared/GameObjects/ContainerTests.cs

```diff
@@ -255,7 +255,7 @@ await server.WaitAssertion(() =>
                  // the entity deleted event is not played on the client if the entity does not exist on the client.
                  if (sEntManager.EntityExists(sItemUid)
                      // && itemUid.TryGetContainer(out var container))
-                     && sContainerSys.TryGetContainingContainer(sItemUid, out var container))
+                     && sContainerSys.TryGetContainingContainer((sItemUid, null), out var container))
```

Comment:
```suggestion
                     && sContainerSys.TryGetContainingContainer(sItemUid, out var container))
```
not needed anymore 😄